include/deal.II/matrix_free/fe_point_evaluation.h

```diff
@@ -2067,11 +2067,7 @@ FEPointEvaluation<n_components_, dim, spacedim, Number>::finish_integrate_fast(
               ETT::write_value(vectorized_input,
                                comp,
                                solution_renumbered_vectorized[i]);
-              for (unsigned int lane = n_lanes_internal / 2; lane > 0;
-                   lane /= 2)
-                for (unsigned int j = 0; j < lane; ++j)
-                  vectorized_input[j] += vectorized_input[lane + j];
-              input[i] = vectorized_input[0];
+              input[i] = vectorized_input.horizontal_add();
```

Comment:
Do you see any difference in your timings by this?

Reply:
I think this might reduce the access to level 1 cache, which is (likely) the bottleneck in this piece of code for low to moderate polynomial degrees.

Reply:
Yes I do, especially for low polynomial degrees, but I also see consistent better timings for higher degrees. For p=1 it is around 5% throughout.